fms/utils/generation.py

```diff
@@ -21,7 +27,8 @@ def _make_cache_contiguous(past_key_value_states):
 
 
 def generate(
-    model: Union[Callable, torch.nn.Module],
+    prefill_model: Union[Callable, torch.nn.Module],
+    decode_model: Union[Callable, torch.nn.Module],
```

Comment:
I think we should make `decode_model` optional and if not set, just set it to `prefill_model`.